installer/manifests/keptn/charts/control-plane/templates/core.yaml

```diff
@@ -643,6 +671,12 @@ spec:
           volumeMounts:
             - mountPath: /data/config
               name: resource-volume
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: {{ .Values.configurationService.fsGroup | default 1001 }}
```

Comment:
I do not think this is correct, we are in resource-service and using values from configuration-service. Also, I am not sure, if we need a security context for the resource-service, as we are using `emptyDir` as a volume instead of `PersistentVolumeClaim` as when using configuration-service. @bacherfl can you maybe support my thoughts ?

Reply:
cross-usage of values is probably not the best idea, but having a security context never hurts IMHO :)

Reply:
I am ok with that, but when adding it, we should add it also here https://github.com/keptn/keptn/blob/master/resource-service/deploy/service.yaml to keep the code consistent :)

Reply:
I am using cross-usage since we want to make sure the switch from configuration- to resource-service works with only the single FF. In the future, when we do the transition and mark the resource-service as production ready and the default, these properties will be renamed together with the other existing ones.

Reply:
Thanks for the explanation!